tools/generate_report.py

```diff
@@ -101,7 +101,7 @@
 with open(dir_report + "/report.md", "w") as f:
     ############################################################################
     # Header
-    f.write("# Package Evaluation Report\n\n")
+    f.write("# Package Evaluation Report for GAP `%s`\n\n" % report["gap_version"])
```

Comment:
Perhaps consider using an "f-string" instead?
```suggestion
    f.write(f"# Package Evaluation Report for GAP `{report['gap_version']}`\n\n")
```

Thatsaid, I'd also be happy to merge this PR as-is, but it is marked as "draft" -- what is it waiting for?

Reply:
I wanted to double-check with a workflow-run in my fork, but then forgot to actually do it. I think it is safe to merge.

Reply:
Seems to work as intended.